qa/tasks/cephfs/test_fstop.py

```diff
@@ -97,7 +97,7 @@ def verify_fstop_metrics(metrics):
                           "--yes-i-really-mean-it")
 
         # create a new filesystem
-        fs_b = self.mds_cluster.newfs(name=newfs_name)
+        fs_b = self.mds_cluster.newfs(name=newfs_name, create=True)
```

Comment:
This has no effect. The `create` kwarg is already defaulting to `True`.

Reply:
right, I was just looking at Filesystem::__init__. Closing PR.